tests/utils.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+
+def get_resource_path(filename):
```

Comment:
You'd probably be okay with using this just in the tests since they're usually not included in the package. I've used pkg_resources when defining the [schemas for topline](https://github.com/mozilla/python_mozetl/blob/master/mozetl/topline/schema.py). It's also being used to define [the main_summary schema](https://github.com/mozilla/python_mozetl/blob/master/mozetl/schemas.py#L12).